### PR TITLE
added networkName to Ytsaurus client

### PR DESCRIPTION
**Pull request by mamogaaa**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

File: yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/YTsaurusClient.java [hunk 1]
```diff
@@ -674,6 +680,17 @@ protected <C, B extends ClientBuilder<C, B>> ClientBuilder<C, B> copyTo(ClientBu
...
             return builder;
         }
+        
+        /**
+         * Set proxy network name to use.
+         *
+         * <p>
+         * If no proxy network name is specified default network name will be used.
+         */
+        public TBuilder setProxyNetworkName(@Nullable String proxyNetworkName) {
```
Comment (tinarsky) [inline yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/YTsaurusClient.java:690]: NIT: this public method should precede the protected `copyTo`.

File: yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/YTsaurusClient.java [hunk 2]
```diff
@@ -674,6 +680,17 @@ protected <C, B extends ClientBuilder<C, B>> ClientBuilder<C, B> copyTo(ClientBu
```
Comment (tinarsky) [inline yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/YTsaurusClient.java:680]: `builder.proxyNetworkName = proxyNetworkName;`

File: yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/YTsaurusClient.java [hunk 3]
```diff
@@ -678,17 +671,6 @@ public TBuilder setProxyRole(@Nullable String proxyRole) {
...
-        /**
-         * Set proxy network name to use.
-         *
-         * <p>
-         * If no proxy network name is specified default network name will be used.
-         */
-        public TBuilder setProxyNetworkName(@Nullable String proxyNetworkName) {
-            this.proxyNetworkName = proxyNetworkName;
-            return self();
-        }
```
Comment (tinarsky) [inline yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/YTsaurusClient.java]: This method should be added here again.

**Comment by tinarsky** [comment]
Shipped!

**Comment by mamogaaa** [comment]
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

**Comment by tinarsky** [comment]
Shipped!

**Pull request closed without merging**